internal/generate_example_overview.py

```diff
@@ -12,7 +12,7 @@ def generate_cards(files: list[str]) -> str:
 
     for file in files:
         path, _ = file.split(".mdx")
-        with open(file, "r") as f:
+        with open(file) as f:
```

Comment:
is this a standard lint rule? seems like explicit is better in situations like this?

Reply:
yeah its an UP rule, as in, it prefers standard practice from newer python versions